lldb/test/API/tools/lldb-dap/progress/TestDAP_Progress.py

```diff
@@ -4,21 +4,55 @@
 
 from lldbsuite.test.decorators import *
 from lldbsuite.test.lldbtest import *
+import json
 import os
 import time
 
 import lldbdap_testcase
 
 
 class TestDAP_progress(lldbdap_testcase.DAPTestCaseBase):
+    def verify_progress_events(
+        self,
+        expected_title,
+        expected_message=None,
+        expected_not_in_message=None,
+        only_verify_first_update=False,
+    ):
+        self.dap_server.wait_for_event("progressEnd", 5)
```

Comment:
If a server is running multiple tests at once, there may be some delays. If you are sending 3 events, one second apart, I would give this timeout a larger value to ensure we don't timeout on busy machines. Something like 10 or 15 seconds at least.